### DISCUSSION TITLE
Open TRANSFORM-Library in OpenModelica

### DISCUSSION THREAD

**Discussion by yifanm4** [ORNL-Modelica/TRANSFORM-Library#102] [Q&A] [answered]
Dear Transform-developers, 

I want to use Transform-library for my research, but I don't have access to Dymola. I was wondering how difficult it is to modify the Transform source codes so it can also work in Openmodelica? Have you guys ever tried it before? 

Thanks!

**Comment by greenwoodms06** [comment, marked as answer]
Good day. There was some testing a while back but found OpenModelica not quite capable enough. Recently OM has made some improvements that, couple with some minor modifications to a few TRANSFORM components, there may be a path forward in using OM for at least some tests. We do have a little bit of effort looking at OM currently that may make some improvements but I don't have any other information on that at the moment.

So in summary, you can try and may find some success however we don't have any recent first hand success using OM we can immediately share. Dymola remains the recommended tool for using TRANSFORM if possible (there are academic rates I believe).

**Reply by yifanm4** [reply]
It's very helpful! Thank you so much
